content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -0,0 +1,332 @@
+
+{{< text bash >}}
+$ cat <<EOF > controlplane-gateway.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+ namespace: istio-system
+spec:
+ components:
+   ingressGateways:
+     - name: istio-ingressgateway
+       enabled: true
+       k8s:
+         service:
+           ports:
+             - port: 15021
+               targetPort: 15021
+               name: status-port
+             - port: 15012
+               targetPort: 15012
+               name: tls-xds
+             - port: 15017
+               targetPort: 15017
+               name: tls-webhook
+EOF
+{{< /text >}}
+
+Apply the configuration in `external_cluster` in the `istio-system` namespace:
+
+{{< text bash >}}
+$ istioctl apply -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+Create the Istio network configuration to expose the **yet to be installed** external
+istiod on the ingress gateway in the `istio-system` namespace:
+
+{{< text bash >}}
+$ cat <<EOF > external-istiod-gw.yaml
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+ name: external-istiod-gw
+ namespace: external-istiod
+spec:
+ selector:
+   istio: ingressgateway
+ servers:
+   - port:
+       number: 15012
+       protocol: https
+       name: https-XDS
+     tls:
+       mode: SIMPLE
+       credentialName: $SSL_SECRET_NAME
+     hosts:
+     - "$REMOTE_ISTIOD_ADDR"
+   - port:
+       number: 15017
+       protocol: https
+       name: https-WEBHOOK
+     tls:
+       mode: SIMPLE
+       credentialName: $SSL_SECRET_NAME
+     hosts:
+     - "$REMOTE_ISTIOD_ADDR"
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+   name: external-istiod-vs
+   namespace: external-istiod
+spec:
+   hosts:
+   - $REMOTE_ISTIOD_ADDR
+   gateways:
+   - external-istiod-gw
+   http:
+   - match:
+     - port: 15012
+     route:
+     - destination:
+         host: istiod.external-istiod.svc.cluster.local
+         port:
+           number: 15012
+   - match:
+     - port: 15017
+     route:
+     - destination:
+         host: istiod.external-istiod.svc.cluster.local
+         port:
+           number: 443
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+ name: external-istiod-dr
+ namespace: external-istiod
+spec:
+ host: istiod.external-istiod.svc.cluster.local
+ trafficPolicy:
+   portLevelSettings:
+   - port:
+       number: 15012
+     tls:
+       mode: SIMPLE
+     connectionPool:
+       http:
+         h2UpgradePolicy: UPGRADE
+   - port:
+       number: 443
+     tls:
+       mode: SIMPLE
+EOF
+{{< /text >}}
+
+Apply the Istio configuration in `external_cluster`:
+
+{{< text bash >}}
+$ kubectl apply -f external-istiod-gw.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+### Setup remote cluster
+
+Generate the Istio configuration for `remote_cluster` and the `external-istiod` namespace:
+
+{{< text bash >}}
+$ cat <<EOF > remote-config-cluster.yaml
```

Comment:
Should specify:

1) the `remote` profile as the base
2) the cluster name
3) the network name (optional?)

Reply:
This should be addressed!  Thanks!  PTAL